dashboard/utils.py

```diff
@@ -523,3 +510,31 @@ def save_event(instance, eligible_attendees):
     # you must save the instance into the database as a row in the table before you can set the manytomany field
     instance.eligible_attendees.set(eligible_attendees)
     instance.save()
+
+
+def get_human_readable_model_name(object):
```

Comment:
What is `object`? should add a docstring here

Reply:
the docstring you added doesnt really line up with `object`

Reply:
I didn't add a docstring to object